packages/showcases/web-components/.storybook/preview-head.html

```diff
@@ -8,7 +8,98 @@
 />
 
 <style>
+  /* Extended toolbar */
+  .extended-toolbar {
+    background-color: var(--vtmn-semantic-color_background-primary);
+    padding: 1rem;
+    border-bottom: 1px solid var(--vtmn-semantic-color_border-secondary);
+    position: fixed;
+    width: 100%;
+    z-index: 99999;
+    top: 0;
+  }
+  .extented-toolbar__title {
+    margin-bottom: 1rem;
+  }
+  .sb-show-main.sb-main-padded {
+    padding: 0;
+  }
+
+  /* Stories' preview */
+  #root {
+    padding: 1rem;
+    margin-top: 150px;
```

Comment:
```suggestion
    margin-top: 180px;
```

Reply:
Tested on snackbar branch, the component can be cut sometimes.

Reply:
Already fixed here: https://github.com/Decathlon/vitamin-web/commit/cd8c64a1242619186ea73c32998cd3d706878396
This bug comes from select container.